output-comparator/src/main/java/org/mobilitydata/gtfsvalidator/outputcomparator/cli/Main.java

```diff
@@ -0,0 +1,188 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.outputcomparator.cli;
+
+import com.beust.jcommander.JCommander;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.flogger.FluentLogger;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+import org.mobilitydata.gtfsvalidator.model.ValidationReport;
+import org.mobilitydata.gtfsvalidator.outputcomparator.io.NoticeStat;
+
+public class Main {
+
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+  private static final String ACCEPTANCE_REPORT_JSON = "acceptance_report.json";
+  private static final int IO_EXCEPTION_EXIT_CODE = 1;
+  private static final int INVALID_NEW_RULE_EXIT_CODE = 2;
+  private static final Gson GSON =
+      new GsonBuilder().serializeNulls().disableHtmlEscaping().create();
+
+  public static void main(String[] argv) {
+    Arguments args = new Arguments();
+    new JCommander(args).parse(argv);
+    List<File> reportDirectory = null;
+    try {
+      reportDirectory =
+          Files.list(Paths.get(args.getReportDirectory()))
+              .map(Path::toFile)
+              .collect(Collectors.toList());
+    } catch (IOException ioException) {
+      ioException.printStackTrace();
+    }
+    if (reportDirectory == null) {
+      logger.atSevere().log("Specified output is not a directory, or an I/O error occurred");
+      return;
+    }
+    if (reportDirectory.isEmpty()) {
+      logger.atSevere().log(
+          "Specified directory is empty, cannot generate acceptance tests report.");
+      return;
+    }
+    int badDatasetCount = 0;
+
+    List<File> reportDirs =
+        reportDirectory.stream().filter(File::isDirectory).collect(Collectors.toList());
+
+    int totalDatasetCount = reportDirs.size();
+    Map<String, NoticeStat> acceptanceTestReportMap = new TreeMap<>();
+
+    try {
+      String urlsAsString = Files.readString(args.getSourceUrlPath());
+      for (File file : reportDirs) {
+        Path referenceReportPath = file.toPath().resolve(args.getReferenceValidationReportName());
+        Path latestReportPath = file.toPath().resolve(args.getLatestValidationReportName());
+        if (!new File(referenceReportPath.toString()).isFile()
```

Comment:
> Why do we want to ignore absence of the files?

We ignore folders that do not contain any validation report such as folders generated by the CI interface of github, otherwise we crash.